WooCommerce/Classes/ViewRelated/Orders/OrderDetailsViewController.swift

```diff
@@ -69,14 +84,18 @@ class OrderDetailsViewController: UIViewController {
         super.viewDidLoad()
         configureNavigation()
         configureTableView()
+        configureTrackingResultsController()
         configureEntityListener()
```

Comment:
You can delete lines 87 and 88 (`configureTrackingResultsController()` and `configureEntityListener()`) since these two funcs are called on L91 & L92